README.md

```diff
@@ -1899,8 +1899,6 @@ Other Style Guides
       : value1 > value2 ? "baz" : null;
 
     // better
-    const maybeNull = value1 > value2 ? 'baz' : null;
```

Comment:
this isn't actually a separate example, it's defining the code used a few lines down. Maybe it'd be better to just remove the empty lines between `const maybeNull` and `const foo`, in both sections?

Reply:
I'd understood the gist but was perplexed when I've found same code in better and good sections) Yes, it makes sense to delete empty lines, but still, same code will be in 2 sections(
Maybe it's possible to consider something like this:
````
// split into 2 separated ternary expression
const maybeNull = value1 > value2 ? 'baz' : null;

// better
const foo = maybe1 > maybe2
  ? 'bar'
  : maybeNull;

// best
const foo = maybe1 > maybe2 ? 'bar' : maybeNull;
````

Reply:
Sure, that's not bad. (`s/expression/expressions` tho)